PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -1110,8 +1093,8 @@ DECLARE_SOA_COLUMN(CTauXic, ctauXic, double);
 DECLARE_SOA_COLUMN(EtaV0PosDau, etaV0PosDau, double);
 DECLARE_SOA_COLUMN(EtaV0NegDau, etaV0NegDau, double);
 DECLARE_SOA_COLUMN(EtaPiFromCasc, etaPiFromCasc, double);
-DECLARE_SOA_COLUMN(EtaPiFromOme, etaPiFromOme, double);
-DECLARE_SOA_COLUMN(EtaOmegac, etaOmegac, double);
+DECLARE_SOA_COLUMN(EtaPiFromCharm, etaPiFromCharm, double);
+DECLARE_SOA_COLUMN(EtaCharm, etaCharm, double);
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(EtaCharmBaryon, etaCharmBaryon, double);
```

This is valid also for other variables that end with `Charm`, which might convince that you are refering to the wuark rather than the hadron